openedx/core/djangoapps/user_api/accounts/__init__.py

```diff
@@ -81,10 +81,10 @@
 
 # Translators: These messages are shown to users who do not enter information
 # into the required field or enter it incorrectly.
-REQUIRED_FIELD_NAME_MSG = _(u"Please enter your Full Name.")
+REQUIRED_FIELD_NAME_MSG = _(u"Enter your full name.")
 REQUIRED_FIELD_CONFIRM_EMAIL_MSG = _(u"The email addresses do not match.")
-REQUIRED_FIELD_COUNTRY_MSG = _(u"Please select your Country.")
-REQUIRED_FIELD_CITY_MSG = _(u"Please enter your City.")
-REQUIRED_FIELD_GOALS_MSG = _(u"Please tell us your goals.")
-REQUIRED_FIELD_LEVEL_OF_EDUCATION_MSG = _(u"Please select your highest level of education completed.")
-REQUIRED_FIELD_MAILING_ADDRESS_MSG = _(u"Please enter your mailing address.")
+REQUIRED_FIELD_COUNTRY_MSG = _(u"Select your country or region of residence.")
+REQUIRED_FIELD_CITY_MSG = _(u"Enter your city.")
+REQUIRED_FIELD_GOALS_MSG = _(u"Tell us your goals.")
+REQUIRED_FIELD_LEVEL_OF_EDUCATION_MSG = _(u"Select your highest level of education completed.")
```

Comment:
@Ayub-Khan Please note this was slightly different than just removing "please": "Select the highest level of education you have completed."